edm4hep.yaml

```diff
@@ -402,6 +402,21 @@ datatypes:
       - edm4hep::Vector3f position [mm] // position of the hit in world coordinates
       - int32_t type                   // type of hit
 
+
+  edm4hep::SimDRCalorimeterHit:
+    Description: "Simulated dual-readout calorimeter hit"
+    Author: "Wonyong Chung"
+    Members:
+      - uint64_t cellID                             // detector cellID
+      - float energy [GeV]                          // energy of the hit
+      - edm4hep::Vector3f position [mm]             // position of the calorimeter cell in world coords
+      - int32_t nCerenkovProd                       // number of cerenkov photons produced
```

Comment:
Why produced photons? Why not recorded photons? Similar to having the avg arrival time of photons recorded in the time fields?

Reply:
This is a SIM level object, I guess we should leave the possibility to treat effects such as efficiency in a subsequent digitizer (though it is indeed sometimes interesting to leak part of the digitization in the SDAction)

Reply:
But then the tAvg should also come from the digitiser and not be calculated here already?

Reply:
Some input from side. Our idea here is to have a first functional simulation which is a sufficiently good representation of the detector performance. The definition of the technology is still being matter of R&D and thus we are not ready yet to work on the digitization part. At this first stage, in our opinion, the most efficient  approach (also from the computational time point of view) is to factorize the simulation of energy deposition part from that of optical photon tracing, SiPM and electronics. 
From the former we need:
- number of cherenkov photons produced (we can then apply offline, at the reconstruction level a scaling on this number which takes into account the light collection efficiency, i.e. light tracing aspects)
- amount of energy deposited in each calorimetric cell

For what concerns the time stamp it is often sufficient to save the average time of all hits inside a certain calorimetric cell (possibly weighed with the energy of each hit). This is for our calorimeter a sufficiently good approximation of the time information we can extract with a realistic system (possibly after applying a resolution smearing).

It is to early to have a realistic implementation of the digitizer step in my opinion because of two reasons:
- we want to explore different configurations and for this it is more convenient to apply "digitization" effects offline
- simulation of optical photons is very time consuming so we are looking into a way to parametrize all optical aspects as much as possible (based on standalone Geant4 studies)

Concerning having separate collections for the scintillation and cherenkov photons this is also possible I believe. However, the difference with respect to the dual-readout fiber calorimeter is that in our case we have a complete duplications of hits since the cherenkov photons are always produced in the same volumes as that of other hits. Having two separate collections looked like redundant (I don't know about the impact on data size). For each hit in a certain calorimetric cell we always have both scintillation and cherenkov non-zero signals. If we use two separate collections we will have to match one-by-one all hits at the reconstruction level.